input/drivers/switch_input.c

```diff
@@ -64,6 +68,20 @@ static void switch_input_poll(void *data)
          sw->touch_y[i] = touch_position.py;
       }
    }
+
+   for (int i = 0; i < SWITCH_NUM_SCANCODES; i++)
```

Comment:
C89 error

Reply:
What is the error? I am not familiar with C89, but I was able to compile without errors using `make -f Makefile.libnx -j12`

Reply:
I made a change...
```
int i = 0;
for (i = 0; ...
```
Is that it?